lacework/resource_lacework_resource_group_azure.go

```diff
@@ -0,0 +1,200 @@
+package lacework
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+	"strings"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkResourceGroupAzure() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkResourceGroupAzureCreate,
+		Read:   resourceLaceworkResourceGroupAzureRead,
+		Update: resourceLaceworkResourceGroupAzureUpdate,
+		Delete: resourceLaceworkResourceGroupAzureDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkResourceGroup,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The resource group name",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "The state of the resource group",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The description of the resource group",
+			},
+			"tenant": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The tenant of the resource group",
```

Comment:
let us make sure we let users know these are id's
```suggestion
				Description: "The Azure tenant id",
```